### ISSUE TITLE
Net4j: support secure websocket connections

### ISSUE THREAD

**Issue by claesrosell** [eclipse-cdo/cdo#25]
This is basically a copy of the old bugzilla issue: https://bugs.eclipse.org/bugs/show_bug.cgi?id=571805

```
Cleartext websockets were implemented with [Bug 561115](https://bugs.eclipse.org/bugs/show_bug.cgi?id=561115).

Adding SSL/TLS encryption to the connection allows for secure setups with remote clients connecting via web.
To support enhanced security with reverse proxy servers, the wss connector shall support setting cookies
for authentication.
```
There is an implementation of wss based on a forked version of CDO4.7 . @mPorhel where looking to move that implementation to CDO 4.22 before a contribution can be made.

**Comment by mPorhel** [comment]
Hi @claesrosell, @estepper, 

I have found some time report our current wss:// implementation for Net4J.
Our internal fork has been moved from CDO 4.7 to CDO 4.22, I might have additional fixes to bring to CDO master corresponding to fixes in other areas than the connectors/acceptors. 

With this list of commits, we currently have working WSS transport in our collaborative layer for Eclipse Sirius in the same way than we add with CDO 4.7.

I will adapt the commits / preferences / comments / implementation after feedback/review.

Regards
